styles/styles.css

```
@@ -0,0 +1,451 @@
+@font-face {
+  font-family:"PT Sans" ;
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format("woff2");
+  font-display: swap;
+}
+@font-face {
+  font-family:"PT Sans" ;
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/ptsans-700.woff2") format("woff2");
+  font-display: swap;
+}
+:root {
+  --main-font: PT Sans, sans-serif;
+}
```

```suggestion
:root {
  --main-font: "PT Sans", sans-serif;
}
```
Не теряй кавычки в значениях с пробелами. Кто-то из браузеров поймет, кто-то решит не понимать.